RealEstate/apps/core/views.py

```diff
@@ -134,37 +133,15 @@ def post(self, request, *args, **kwargs):
         """
         homebuyer = Homebuyer.objects.filter(user_id=request.user.id)
         couple = Couple.objects.filter(homebuyer__user=request.user)
-        categories = Category.objects.filter(couple=couple)
         house = get_object_or_404(House.objects.filter(id=kwargs["house_id"]))
 
-        default_score = Grade._meta.get_field('score').default
-        for category in categories:
-            value = request.POST.get(str(category.id)) or default_score
-            grade, created = Grade.objects.update_or_create(
-                homebuyer=homebuyer.first(), category=category, house=house, defaults={'score': int(value)})
-
-        grades = Grade.objects.filter(house=house, homebuyer=homebuyer)
-
-        # Merging grades and categories to provide object with both information.
-        # Data Structure: [(cat1, score1), (cat2, score2), ...]
-        graded = []
-        for category in categories:
-            missing = True
-            for grade in grades:
-                if grade.category.id is category.id:
-                    graded.append((category, grade.score))
-                    missing = False
-                    break
-            if missing:
-                graded.append((category, None))
-        form = EvaluationForm(graded=graded)
-
-        messages.success(request,"Your evaluation was saved!")
-        context = {
-            'couple': couple,
-            'house' : house,
-            'grades': graded,
-            'form' : form
-        }
-        context.update(self._score_context())
-        return render(request, self.template_name, context)
+        if request.is_ajax():
+          id = request.POST['category']
+          score = request.POST['score']
+          category = Category.objects.get(id=id)
+          grade, created = Grade.objects.update_or_create(
+                homebuyer=homebuyer.first(), category=category, house=house, defaults={'score': int(score)})
+          response_data = {}
+          response_data['id'] = str(id)
+          response_data['score'] = str(score)
```

Comment:
Condensable to a single statement.  Doesnt really change the behavior so this can just be programmer preference.
```python
response_data = {
    'id': str(id),
    'score': str(score)
}
```